src/renderers/WebGLRenderer.js

```diff
@@ -1147,6 +1151,61 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	}
 
+	this.renderOverride = function ( overrideMaterial, scene, camera, renderTarget, clearColor, clearAlpha ) {
+
+		var originalClearColor = this.getClearColor(), originalClearAlpha = this.getClearAlpha(), originalAutoClear = this.autoClear;
+
+		this.autoClear = false;
+		clearColor = overrideMaterial.clearColor || clearColor;
+		clearAlpha = overrideMaterial.clearAlpha || clearAlpha;
+		var clearNeeded = ( clearColor !== undefined )&&( clearColor !== null );
+		if( clearNeeded ) {
+			this.setClearColor( clearColor );
+ 	 		this.setClearAlpha( clearAlpha || 0.0 );
+		}
+
+		scene.overrideMaterial = overrideMaterial;
+		this.render( scene, camera, renderTarget, clearNeeded );
+		scene.overrideMaterial = null;
+
+		// restore original state
+		this.autoClear = originalAutoClear;
+		this.setClearColor( originalClearColor );
+		this.setClearAlpha( originalClearAlpha );
+
+	}
+
+
+	this.renderPass = function ( passMaterial, renderTarget, clearColor, clearAlpha ) {
+
+		if( ! this.passScene ) {
+			this.passCamera = new THREE.OrthographicCamera(-1, 1, 1, -1, 0, 1);
+			this.passQuad = new THREE.Mesh(new THREE.PlaneGeometry(2, 2), null);
+			this.passScene = new THREE.Scene();
+			this.passScene.add( this.passQuad );
+		}
+
+		// save original state
+		var originalClearColor = this.getClearColor(), originalClearAlpha = this.getClearAlpha(), originalAutoClear = this.autoClear;
+
+		// setup pass state
+		this.autoClear = false;
+		var clearNeeded = ( clearColor !== undefined )&&( clearColor !== null );
+		if( clearNeeded  ) {
+			this.setClearColor( clearColor );
+			this.setClearAlpha( clearAlpha || 0.0 );
+		}
+
+		this.passQuad.material = passMaterial;
+		this.render( this.passScene, this.passCamera, renderTarget, clearNeeded );
```

Comment:
I think you should be able to do `this.render( this.passQuad, this.pass camera )` directly.